include/verilated_vcd_c.h

```diff
@@ -126,6 +126,7 @@ class VerilatedVcd VL_NOT_FINAL : public VerilatedTrace<VerilatedVcd, VerilatedV
     // ACCESSORS
     // Set size in megabytes after which new file should be created
     void rolloverMB(uint64_t rolloverMB) { m_rolloverMB = rolloverMB; }
+    uint64_t getWroteBytes() { return m_wroteBytes; }
```

Comment:
```suggestion
    uint64_t wroteBytes() { return m_wroteBytes; }
```

Reply:
Actually should be const.  And if we add this function we'll need a test to (roughly) check it.

Reply:
Cool, sounds good. I'll try to figure out how to add it and get it done.